internal/analytics/analytics.go

```diff
@@ -207,6 +226,8 @@ func Configure(cfg configurable) {
 
 // Event logs an event to google analytics
 func Event(category string, action string) {
+	category = appPrefix + category
```

Comment:
This is prefixing every single category with the appPrefix. What we want is to prefix the event (not category) with the prefix ONLY for the following events:

- run-command
- command-exit

This logic should not live in the analytics package. It should live where those events are being fired.